akka-actor/src/main/scala/akka/dispatch/Mailbox.scala

```diff
@@ -232,7 +232,7 @@ private[akka] abstract class Mailbox(val messageQueue: MessageQueue)
       }
     } finally {
       setAsIdle() //Volatile write, needed here
-      dispatcher.registerForExecution(this, false, false)
+      dispatcher.registerForExecution(this, false, false, true)
```

Comment:
This is the magic improvement where we say that we want to "reschedule" the mailbox, i.e. we yield because this actor has reached its throughput.